modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationService.java

```diff
@@ -18,6 +18,10 @@
  */
 package org.nuxeo.runtime.migration;
 
+import java.util.List;
+
+import javax.resource.spi.IllegalStateException;
```

Comment:
You shouldn't use this exception from `javax.resource.spi`.

Reply:
I'm a bit lost about what to use as I can't access NuxeoException. I put RuntimeException wdyt ?

Reply:
`java.lang.IllegalStateException` would be ok. Or use `org.nuxeo.runtime.RuntimeServiceException` if you want a nuxeo-specific exception but not for the signature as it's too generic.

Reply:
If no caller is really expected to catch the exception and do special treatment, don't do anything in the signature and just throw `RuntimeServiceException` or `IllegalStateException` from the code in question.

Reply:
Thanks!